Kernel/Process.cpp

```diff
@@ -2958,7 +2959,8 @@ int Process::sys$select(const Syscall::SC_select_params* params)
     fd_set* readfds = params->readfds;
     fd_set* writefds = params->writefds;
     fd_set* exceptfds = params->exceptfds;
-    timeval* timeout = params->timeout;
+    const timespec* timeout = params->timeout;
+    const sigset_t *sigmask = params->sigmask;
```

Comment:
```suggestion
    const sigset_t* sigmask = params->sigmask;
```

Reply:
I ran `git rebase -i HEAD~3`, picked 'edit' for each commit, and ran `git clang-format --diff HEAD^; git commit -a --amend --no-edit; git rebase --continue` for each of the 3 commits. Is there some nicer way to do this?

Reply:
I would try `git rebase --exec` or even `git filter-branch`. Or simpler, just run clang-format once, then commit the changes, splitting them into different commits for different parts of code, then squash those commits into the originals (may even use autosquash functionality for this).

Reply:
I have `clang-format` bound to a keyboard shortcut in my editor and press it compulsively. It works pretty well :)